app/notify_client/service_api_client.py

```diff
@@ -348,23 +348,23 @@ def get_most_recent_inbound_number_usage_date(self, service_id):
         )
 
     @cache.delete("service-{service_id}")
-    def create_service_inbound_api(self, service_id, url, bearer_token, user_id):
-        data = {"url": url, "bearer_token": bearer_token, "updated_by_id": user_id}
+    def create_service_inbound_api(self, service_id, url, bearer_token, user_id, callback_type):
+        data = {"url": url, "bearer_token": bearer_token, "updated_by_id": user_id, "callback_type": callback_type}
         return self.post(f"/service/{service_id}/inbound-api", data)
```

Comment:
Can start using the new, generic URL as part of this PR:
```suggestion
        return self.post(f"/service/{service_id}/callback", data)
```

Same comment applies to all relevant client methods

Reply:
could we though? Since the merge order is:

1. https://github.com/alphagov/notifications-api/pull/4405 (just schema changes)
2. This PR - sending types to API
3. https://github.com/alphagov/notifications-api/pull/4402 - actually changing the endpoints in the API

Reply:
You’re right can’t start this yet

Reply:
It can be done either post table merge or alongside the update to the api to write to both callback tables .

Reply:
ok, when [the last commit](https://github.com/alphagov/notifications-admin/pull/5417/commits/85303b3894f6dfe201b0ff36a2b8a9be7d2d5926) is removed, I am happy to approve then

Reply:
> ok, when [the last commit](https://github.com/alphagov/notifications-admin/pull/5417/commits/85303b3894f6dfe201b0ff36a2b8a9be7d2d5926) is removed, I am happy to approve then

Done